remote-api/src/main/java/org/alfresco/rest/api/impl/rules/RuleSetsImpl.java

```diff
@@ -65,7 +66,32 @@ public RuleSet getRuleSetById(String folderNodeId, String ruleSetId, List<String
         NodeRef folderNode = validator.validateFolderNode(folderNodeId, false);
         NodeRef ruleSetNode = validator.validateRuleSetNode(ruleSetId, folderNode);
 
-        return RuleSet.of(ruleSetNode.getId());
+        return loadRuleSet(ruleSetNode, includes);
+    }
+
+    /**
+     * Load a rule set for the given node ref.
+     *
+     * @param ruleSetNodeRef The rule set node.
+     * @param includes A list of fields to include.
+     * @return The rule set object.
+     */
+    protected RuleSet loadRuleSet(NodeRef ruleSetNodeRef, List<String> includes)

```

Comment:
Why is this method `protected`? I assume (but may be wrong) that it should be `private` and additional unit test should be hitting `getRuleSetById` and `getRuleSets`  methods (just with different set of input parameters).

Reply:
The method is protected to allow unit testing it. I expect this method to grow as we add support for more fields, and so I think it makes sense to test it in isolation.  I could move it to a new class instead?